index.js

```diff
@@ -60,9 +60,7 @@ function Option(flags, description) {
  */
 
 Option.prototype.name = function() {
-  return this.long
-    .replace('--', '')
-    .replace('no-', '');
+  return this.long.replace(/^--/, '');
```

Comment:
Note: This may be a breaking change for people who use the (only recently-documented) [Custom Event Listeners](https://github.com/tj/commander.js#custom-event-listeners), though I can also clarify this by adding another illustrative example to the Readme in that section, if required.